core/src/main/java/org/commonjava/maven/ext/manip/ManipulationManager.java

```diff
@@ -150,6 +156,32 @@ public void scanAndApply( final ManipulationSession session )
             }
         }
 
+        applyAfterManipulations( projects, session );
+
+        try
+        {
+            File resultFile = new File( session.getTargetDir(), ManipulationManager.RESULT_FILE );
+            boolean recreated = true;
+            resultFile.getParentFile().mkdirs();
+            if ( resultFile.exists() )
+            {
+                recreated = recreated && resultFile.delete();
+            }
```

Comment:
Won't filewriter overwrite making this deletion code redundant. If this code was part of the marker file block above it would also make the mkdirs redundant.